### PR TITLE
Various fixes, including Revert "feat(api): look up nslord address only once"

### PR DESCRIPTION
**Pull request by peterthomassen**
This reverts commit 1c6a6878708e43a5eba0bbf699219f33a4ad9c57.

### CODE REVIEW

**Comment by nils-wisiol** [comment]
lgtm

**Review by nils-wisiol — APPROVED** [review]

File: www/webapp/src/views/Home.vue [hunk 1]
```diff
@@ -354,6 +354,13 @@ export default {
                + ' bulk operations. It is <a href="https://desec.readthedocs.io/en/latest/dns/domains.html" target="_blank">well-documented</a> '
                + ' and easily integrates into your scripts, tools, or CI/CD pipeline.',
       },
+      {
+        href: '#',
+        icon: 'mdi-two-factor-authentication',
+        title: 'Multi-Factor Auth (2FA)',
+        text: 'Accidentally shared your password with someone? Enable MFA to <b>keep your account remains safe</b>. '
```
Comment (nils-wisiol) [inline www/webapp/src/views/Home.vue:361]: Grammar

**Review by nils-wisiol — APPROVED** [review]
only nits

>  fix(api): avoid nslord lookup on load (breaks Celery) 

sounds like this commit breaks things

File: www/webapp/src/views/Home.vue (hunk 1 above)
Comment (peterthomassen) [reply www/webapp/src/views/Home.vue]: done

**Pull request merged**